xsd/netex-nl-data.xsd

```diff
@@ -1103,6 +1103,7 @@ Bij een Version is attribuut 'version' verplicht. De waarde is gelijk aan attrib
 	</xsd:complexType>
 	<xsd:complexType name="passengerCapacity">
 		<xsd:sequence>
+			<xsd:attribute name="id" type="PassengerCapacityIdType" use="optional">
```

Comment:
attribuut buiten de sequence, niet er in

Reply:
Aangepast, PassengerCapacities_RelStructure van CEN XSD overgenomen.